Have "local ephemeral SSD" point to the Temporary Disk documentation. (#8822)
MicrosoftDocs/sql-docs: azure-sql/includes/virtual-machines-best-practices-storage.md

  - For the log drive plan for capacity and test performance versus cost while evaluating the [premium P30 - P80 disks](/azure/virtual-machines/disks-types#premium-ssds)
    - If submillisecond storage latency is required, use [Azure ultra disks](/azure/virtual-machines/disks-types#ultra-disks) for the transaction log.
    - For M-series virtual machine deployments consider [write accelerator](/azure/virtual-machines/how-to-enable-write-accelerator) over using Azure ultra disks.

>   - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the temporary disk (the temporary disk is ephemeral, and defaults to D:\) for most SQL Server workloads that aren’t part of Failover Cluster Instance (FCI) after choosing the optimal VM size.

Reviewer: ```suggestion
  - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the temporary disk (the temporary disk is ephemeral, and defaults to D:\) for most SQL Server workloads that aren’t part of a failover cluster instance (FCI) after choosing the optimal VM size.
```

>   - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the [local ephemeral SSD](azure/virtual-machines/managed-disks-overview#temporary-disk) (default `D:\`) drive for most SQL Server workloads that aren't part of Failover Cluster Instance (FCI) after choosing the optimal VM size.

Reviewer 2: ```suggestion
  - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the [local ephemeral SSD](/azure/virtual-machines/managed-disks-overview#temporary-disk) (default `D:\`) drive for most SQL Server workloads that aren't part of Failover Cluster Instance (FCI) after choosing the optimal VM size.
```

>   - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the [local ephemeral SSD](/azure/virtual-machines/managed-disks-overview#temporary-disk) (default `D:\`) drive for most SQL Server workloads that aren't part of Failover Cluster Instance (FCI) after choosing the optimal VM size.

Reviewer: ```suggestion
  - Place [tempdb](/sql/relational-databases/databases/tempdb-database) on the temporary disk (the temporary disk is ephemeral, and defaults to D:\) for most SQL Server workloads that aren’t part of Failover Cluster Instance (FCI) after choosing the optimal VM size.
```
